### ISSUE TITLE
[1.19.3]: Cannot adjust surface max distance

### ISSUE THREAD

**Issue by Epokhe** [TeamJM/journeymap#578]
### What happened?

I am trying to reduce the distance that Journeymap renders on the map by adjusting "Surface Max Distance" value on Cartography settings. For example, my render distance in video settings is 12, but I want Journeymap to render 2 chunks max. But I can't move the slider to values between 0 and 12, except 9.  I can only move the slider to value 9(I don't know why 9 specifically). The other values than 9 show but don't get saved, I can verify when I re-open the option panel.

I also had a problem with the server setting "Force Map Render Range Max". I set that to 2, but Journeymap continues to render more than 2 chunks. Since that didn't work, I tried the same thing with the client setting, and discovered the problem above. Mentioning this in case two problems are related.

I am using fabric loader 0.14.14.
The issue happens in both 1.19.3 with JM 5.9.4 and 1.19.4 with JM 5.9.4beta2.

Link to the discord chat about the bug report:
https://discord.com/channels/239040424214396929/1088558440227536976

### Minecraft Version

1.19.3

### Version

5.9.4 (Default)

### Relevant log output

Logs and screenshots are uploaded in the discord chat:
https://discord.com/channels/239040424214396929/1088558440227536976

**Comment by mysticdrew** [comment]
This is fixed for next release. 
One note, a work around would be to change the config file manually until the fix is published. 
You can modify the config value
`./minecraft/journeymap/config/5.9/journeymap.core.config` open it your favorite text editor and update the follow fields to the values you want. This manual edit will require a minecraft restart. 
```
  "renderDistanceCaveMax": "0",
  "renderDistanceSurfaceMax": "4",
```

I am not sure when this bug fix release will be published. I cannot promise a 1.19.3 release, as I have officially dropped support for it. Depends if more major bugs crop up.

**Comment by mysticdrew** [comment]
will close this when I release the fix

**Comment by mysticdrew** [comment]
5.9.5 is released on CF and Modrinth for 1.19.3
